jepsen/src/jepsen/control/util.clj

```diff
@@ -104,20 +118,25 @@
 
   Options:
 
-    :force?     Even if we have this cached, download the tarball again anyway."
+    force?      Even if we have this cached, download the tarball again anyway.
```

Comment:
You might want to keep the leading colons here, just so folks know they're keywords rather than symbols. Not a huge deal. :)